### PR TITLE
Remove script-loader

### PR DESCRIPTION
**Pull request by davidmehren**
### Component/Part
Webpack config

### Description
[script-loader](https://github.com/webpack-contrib/script-loader) is deprecated since 2 years and a very ugly way of loading libraries. This PR removes script-loader and replaces them with supported webpack loaders.
It also replaces our vendored dependencies with upstream versions on NPM where appropriate (meaning when our version does not look patched).

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -142,6 +142,7 @@
     "url": "https://github.com/hedgedoc/hedgedoc.git"
   },
   "devDependencies": {
+    "abcjs": "^5.12.0",
```
Comment (mrdrogdrog) [inline package.json:145]: Pin

File: webpack.common.js [hunk 2]
```diff
@@ -386,84 +346,113 @@ module.exports = {
...
+        test: /\.less$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1
+            }
+          },
+          'less-loader'
```
Comment (mrdrogdrog) [inline webpack.common.js:409]: 😱 why less?

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]

File: webpack.common.js (hunk 2 above)
Comment (davidmehren) [reply webpack.common.js:409]: We used that before :man_shrugging:

Comment (mrdrogdrog) [reply webpack.common.js:409]: Do we even have less code?...

File: package.json (hunk 1 above)
Comment (davidmehren) [reply package.json]: done

File: webpack.common.js (hunk 2 above)
Comment (davidmehren) [reply webpack.common.js:409]: Bootstrap and fork-awesome seem to use it

Comment (mrdrogdrog) [reply webpack.common.js:409]: Okay.

**Review by SISheogorath — APPROVED** [review]

**Review by InnayTool — APPROVED** [review]

**Comment by SISheogorath** [comment]
👏🏻 Just amazing!

**Review by mrdrogdrog — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**